pandas/tests/indexes/period/test_arithmetic.py

```diff
@@ -11,6 +11,166 @@
 import pandas.core.indexes.period as period
 
 
+class TestPeriodIndexComparisons(object):
+    @pytest.mark.parametrize('freq', ['M', '2M', '3M'])
+    def test_pi_pi_comp(self, freq):
+        base = PeriodIndex(['2011-01', '2011-02', '2011-03', '2011-04'],
+                           freq=freq)
+        per = Period('2011-02', freq=freq)
+
+        exp = np.array([False, True, False, False])
+        tm.assert_numpy_array_equal(base == per, exp)
+        tm.assert_numpy_array_equal(per == base, exp)
+
+        exp = np.array([True, False, True, True])
+        tm.assert_numpy_array_equal(base != per, exp)
+        tm.assert_numpy_array_equal(per != base, exp)
+
+        exp = np.array([False, False, True, True])
+        tm.assert_numpy_array_equal(base > per, exp)
```

Comment:
can u parametrize on Period and PI
and make errors in a separate test

Reply:
Did some of this, leaving some of the parameterization for the next pass, since test_comp_nat is already a mess

Reply:
This is turning out to be a PITA.  Currently parametrizing _other_ tests in the same file to earn goodwill.